homework1/task03.py

```diff
@@ -0,0 +1,28 @@
+"""
+Write down the function, which reads input line-by-line, and find maximum and minimum values.
+Function should return a tuple with the max and min values.
+
+For example for [1, 2, 3, 4, 5], function should return [1, 5]
+
+We guarantee, that file exists and contains line-delimited integers.
+
+To read file line-by-line you can use this snippet:
+
+with open("some_file.txt") as fi:
+    for line in fi:
+        ...
+
+"""
+from typing import Tuple
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
+    values = []
+    with open(file_name) as fi:
+        for line in fi:
+            values.append(int(line))
+
+    min_element = min(values)
+    max_element = max(values)
+    res = (min_element, max_element)
```

Comment:
"res" is redundant variable here, it may be replaced by simple 
`return min_element, max_element`